rbx_binary/src/serializer.rs

```diff
@@ -696,6 +704,33 @@ impl<'a, W: Write> BinarySerializer<'a, W> {
 
                         chunk.write_interleaved_u32_array(&buf)?;
                     }
+                    Type::Color3uint8 => {
+                        let mut r = Vec::with_capacity(values.len());
+                        let mut g = Vec::with_capacity(values.len());
+                        let mut b = Vec::with_capacity(values.len());
+
+                        for (i, rbx_value) in values {
+                            if let Variant::Color3uint8(value) = rbx_value.as_ref() {
+                                r.push(value.r);
+                                g.push(value.g);
+                                b.push(value.b);
+                            } else {
```

Comment:
We should probably also support `Color3` values here, since they're often trivially converted. If the `Color3`'s range does not fit within a `Color3uint8`, it should be clamped.